### ISSUE TITLE
/usr/local/bin/nyuu contains DOS-style CR-LF => /usr/bin/env: ‘node\r’: No such file or directory

### ISSUE THREAD

**Issue by sanderjo** [animetosho/Nyuu#22]
I installed nyuu via
```
sudo npm install -g nyuu
```
Trying to start, results in:
```
sander@Stream-13:~$ nyuu
/usr/bin/env: ‘node\r’: No such file or directory
```
Inspecting /usr/local/bin/nyuu shows it contains DOS-style CR-LF's. See below.

Solved with `dos2unix`:

```
sander@Stream-13:~$ ll /usr/local/bin/nyuu
lrwxrwxrwx 1 root root 36 jun 22 23:39 /usr/local/bin/nyuu -> ../lib/node_modules/nyuu/bin/nyuu.js*

sander@Stream-13:~$ sudo cp /usr/local/lib/node_modules/nyuu/bin/nyuu.js /usr/local/lib/node_modules/nyuu/bin/nyuu.js.org

sander@Stream-13:~$ sudo dos2unix /usr/local/lib/node_modules/nyuu/bin/nyuu.js
dos2unix: converting file /usr/local/lib/node_modules/nyuu/bin/nyuu.js to Unix format...

sander@Stream-13:~$ nyuu 
Must supply at least one input file
Enter `nyuu --help` or `nyuu --help-full` for usage information
```
Solved!

Before the repair:
```
sander@Stream-13:~$ which nyuu
/usr/local/bin/nyuu


sander@Stream-13:~$ /usr/local/bin/nyuu
/usr/bin/env: ‘node\r’: No such file or directory

sander@Stream-13:~$ 
sander@Stream-13:~$ hd /usr/local/bin/nyuu | head -20
00000000  23 21 2f 75 73 72 2f 62  69 6e 2f 65 6e 76 20 6e  |#!/usr/bin/env n|
00000010  6f 64 65 0d 0a 0d 0a 22  75 73 65 20 73 74 72 69  |ode...."use stri|
00000020  63 74 22 3b 0d 0a 70 72  6f 63 65 73 73 2e 74 69  |ct";..process.ti|
00000030  74 6c 65 20 3d 20 27 4e  79 75 75 27 3b 0d 0a 0d  |tle = 'Nyuu';...|
00000040  0a 76 61 72 20 66 73 20  3d 20 72 65 71 75 69 72  |.var fs = requir|
00000050  65 28 27 66 73 27 29 3b  0d 0a 76 61 72 20 75 74  |e('fs');..var ut|
00000060  69 6c 20 3d 20 72 65 71  75 69 72 65 28 27 2e 2e  |il = require('..|
00000070  2f 6c 69 62 2f 75 74 69  6c 27 29 3b 0d 0a 76 61  |/lib/util');..va|
00000080  72 20 65 72 72 6f 72 20  3d 20 66 75 6e 63 74 69  |r error = functi|
00000090  6f 6e 28 6d 73 67 29 20  7b 0d 0a 09 63 6f 6e 73  |on(msg) {...cons|
000000a0  6f 6c 65 2e 65 72 72 6f  72 28 6d 73 67 29 3b 0d  |ole.error(msg);.|
000000b0  0a 09 63 6f 6e 73 6f 6c  65 2e 65 72 72 6f 72 28  |..console.error(|
000000c0  27 45 6e 74 65 72 20 60  6e 79 75 75 20 2d 2d 68  |'Enter `nyuu --h|
000000d0  65 6c 70 60 20 6f 72 20  60 6e 79 75 75 20 2d 2d  |elp` or `nyuu --|
000000e0  68 65 6c 70 2d 66 75 6c  6c 60 20 66 6f 72 20 75  |help-full` for u|
000000f0  73 61 67 65 20 69 6e 66  6f 72 6d 61 74 69 6f 6e  |sage information|
00000100  27 29 3b 0d 0a 09 70 72  6f 63 65 73 73 2e 65 78  |');...process.ex|
00000110  69 74 28 31 29 3b 0d 0a  7d 3b 0d 0a 76 61 72 20  |it(1);..};..var |
00000120  70 72 6f 63 65 73 73 65  73 3b 0d 0a 76 61 72 20  |processes;..var |
00000130  70 72 6f 63 65 73 73 53  74 61 72 74 20 3d 20 66  |processStart = f|
```

See the `oa 0d` sequences ...

**Comment by scriptzteam** [comment]
month ago bro months ago xD https://github.com/animetosho/Nyuu/issues/14

**Comment by animetosho** [comment]
Geh, forgot about NPM, thanks for the report.

I can't edit NPM easily unfortunately, I'll have to push out a new release to update it. Git doesn't have the problem as it converts newlines properly. For now, your solution seems to be the best.

**Comment by animetosho** [comment]
Should now be fixed in v0.3.3, which is also on NPM.

Thanks again for the report!

**Comment by sanderjo** [comment]
I checked, and it indeed works. Thank you.
